modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/PlatformTestNodeRunner.java

```diff
@@ -73,12 +73,12 @@ public class PlatformTestNodeRunner {
     private static final String TABLE_NAME_ALL_COLUMNS_SQL = "tbl_all_columns_sql"; // All column types supported by SQL.
 
     /** Time to keep the node alive. */
-    private static final int RUN_TIME_MINUTES = 30;
+    private static final int RUN_TIME_MINUTES = 300;
```

Comment:
The idea behind 30 minutes is to deal with hanging processes on TeamCity - the process will exit by itself after some time. Not sure if 300 minutes makes any sense.

Reply:
It was for the debugging as well :) Reversing it.